contracts/priv/cloud-priv.yml

```diff
@@ -708,6 +885,22 @@ components:
         - domain
         - redirectURL
         - state
+    Identity:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+    IdentityLinks:
```

Comment:
#optional Maybe `IdentityWithLinks` instead for clarity? In the code above, when I saw `IdentityLinks` I was wondering why it was only the links and not the identity as well.

Reply:
It's with one or the other. An identity with the links, or one without. I've changed it to not have the links. Let me know if this is not what you want because I have to update the Issue with the changed API spec and change my code as well